app/uk/gov/hmrc/minorentityidentificationfrontend/utils/WritesForJourneyEnd.scala

```diff
@@ -0,0 +1,35 @@
+/*
```

Comment:
This feels a bit oddly located to me as the other "writesForJourney" methods are defined in objects in the models package. Perhaps it could be defined in a trait in the models package.

Reply:
It is not critical, so we can leave it here unless someone else complains.